README.md

```diff
@@ -1 +1,67 @@
-# my-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Dyary's Portfolio
+
+> this is my portfolio website. It includes info about me, how to contact me and my achivemets and creations as a web developer.
+
```

Comment:
I know you did a good effort doing this description for your project, but just try to make a more descriptive README file, if you want to have a reference, three lines or more would be nice, just try to think about what you did, what kind of technologies or techniques you used, and so on. Keep in mind this for this and your next projects 😉 

- [ ] Improve description for your project.